aws/resource_aws_ec2_fleet_test.go

```diff
@@ -1756,6 +1785,33 @@ resource "aws_ec2_fleet" "test" {
 `, allocationStrategy)
 }
 
+func testAccAWSEc2FleetConfig_SpotOptions_CapacityRebalance(rName, allocationStrategy string) string {
+	return testAccAWSEc2FleetConfig_BaseLaunchTemplate(rName) + fmt.Sprintf(`
+resource "aws_ec2_fleet" "test" {
+  launch_template_config {
+    launch_template_specification {
+      launch_template_id = aws_launch_template.test.id
+      version            = aws_launch_template.test.latest_version
+    }
+  }
+
+  spot_options {
+    allocation_strategy = %[1]q
+    maintenance_strategies {
+      capacity_rebalance {
+        replacement_strategy = "launch"
+      }
+    }	
```

Comment:
Extra whitespace at EOL.

```suggestion
    }
```

Reply:
Removed unneeded spacing.